.github/workflows/ci-prerelease.yml

```diff
@@ -70,3 +70,22 @@ jobs:
       AWS_S3_BUCKET_NAME: "nr-downloads-ohai-testing"
       AWS_S3_LOCK_BUCKET_NAME: "onhost-ci-lock-testing"
       ASSETS_TYPE: "all"
+
+  packaging-tests:
+    runs-on: ubuntu-20.04
+    needs: [publishing-to-s3]
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Set and validate distribution name and go-version
+        run: .github/workflows/scripts/set_version.sh
```

Comment:
set_version is executed on pre-release step. Is it necessary here ☝️ ?

Reply:
I think so, `packaging-tests` is another job (it could run on another instance) so environment variables set on the previous jobs do not persistent. I found this issue on the initial commints: https://github.com/newrelic/opentelemetry-collector-releases/actions/runs/4024935623/workflow